docs/section22/Rule22-4.md

```diff
@@ -0,0 +1,58 @@
+
+# CHW&CW - Rule 22-4  
+
+**Rule ID:** 22-4  
+**Rule Description:** For Baseline chilled water loop that is not purchased chilled water and does not serve any computer room HVAC systems, chilled-water supply temperature shall be reset using the following schedule: 44F at outdoor dry-bulb temperature of 80F and above, 54F at 60F and below, and ramped linearly between 44F and 54F at temperature between 80F and 60F.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule 22-3.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, or 13: `PLACEHOLDER`
+
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+3. Pass Rule 22-3: `if Rule-22-3 == "PASS"`
+
```

Comment:
Not sure this is needed.

Reply:
Rule 22-3 checks if the loop temperature_reset_type is "OUTSIDE_AIR_RESET". This rule checks the OSA reset schedule. Can we please discuss if a PASS for Rule 22-3 is needed as applicability check in this case.

Reply:
it might make sense to have 22-3 = Pass for 22-4 applicability check. Lets discuss tomorrow.

Reply:
@xingcx should like 22 is if Rule-22-3 == 'Pass'. it currently says if Rule-22-4 =='Pass'